aict_tools/preprocessing.py

```diff
@@ -37,25 +37,20 @@ def check_valid_rows(df):
 
 def calc_true_disp(source_x, source_y, cog_x, cog_y, delta, project_disp=False):
     ''' Calculate the training variables for the disp regressor '''
-    if project_disp is False:
-        true_disp = euclidean_distance(
-            source_x, source_y,
-            cog_x, cog_y
-        )
-
-        true_delta = np.arctan2(
-            cog_y - source_y,
-            cog_x - source_x,
-        )
-        true_sign = np.sign(np.abs(delta - true_delta) - np.pi / 2)
```

Comment:
So this saved us before?

Reply:
Yes, basically it was 180degrees difference to the usual definition of the sign